registry/registry.go

```diff
@@ -45,7 +45,7 @@ type RouteRegistry struct {
 	pruneStaleDropletsInterval time.Duration
 	dropletStaleThreshold      time.Duration
 
-	reporter metrics.RouteRegistryReporter
+	multiReporter metrics.MultiRouteRegistryReporter
```

Comment:
Couldn't this type stay the interface type it was previously? The coding here should not need to know what reporter implementation is passed to it.